actors/builtin/power/power_test.go

```diff
@@ -609,15 +609,24 @@ func TestCron(t *testing.T) {
 
 		rt.SetEpoch(1)
 		actor.enrollCronEvent(rt, miner1, 2, []byte{0x1, 0x3})
-		actor.enrollCronEvent(rt, miner2, 4, []byte{0x2, 0x3})
+		actor.enrollCronEvent(rt, miner2, 4, []byte{0x1, 0x3})
```

Comment:
checking ordering is an explict intent of this test iiuc.  The test checks that the first enrolled event gets processed then the next by setting two different byte values.  To maintain this testing property you should change this bad to 0x2